pom.xml

```diff
@@ -22,20 +22,14 @@
       </license>
    </licenses>
 
-   <developers>
-      <developer>
-         <name>Weld committers</name>
-      </developer>
-   </developers>
-
    <properties>
       <!-- Force encoding to UTF-8 (bug on OS X) -->
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
       <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
       <pdf.name>${project.artifactId}.pdf</pdf.name>
-      <minimum.maven.version>3.0.3</minimum.maven.version>
-      <maven.compiler.source>1.6</maven.compiler.source>
-      <maven.compiler.target>1.6</maven.compiler.target>
+      <minimum.maven.version>3.2.5</minimum.maven.version>
+      <maven.compiler.source>1.7</maven.compiler.source>
+      <maven.compiler.target>1.7</maven.compiler.target>
```

Comment:
I am not sure we want/need to increase these versions, but this file should be used in Weld 2.x and 3.x which both require Java 7.
As for Maven, 3.2.5 is the last version of Maven which works with Java 6. I don't think we need to enforce anything more up to date. OTOH I am not sure 3.0.3 is safe for all the plugin updates made here.